src/generic/scrlwing.cpp

```diff
@@ -550,6 +555,10 @@ void wxScrollHelperBase::HandleOnScroll(wxScrollWinEvent& event)
         m_win->Refresh(true, GetScrollRect());
     }
 #endif // __WXUNIVERSAL__
+
+    if (m_win) {
+        m_win->UnRef();
+    }
```

Comment:
And whatever we use for the flag we need a RAII helper (e.g. a scope guard) to ensure that it's reset on function exit instead of doing it manually here.

Reply:
Completely agree on that remark.